File: android/LottieInL/app/src/main/java/com/gmail/htaihm/myapplication/MainActivity.java
package com.gmail.htaihm.myapplication;

import android.util.Log;
import androidx.appcompat.app.AppCompatActivity;
import android.os.Bundle;
import com.airbnb.lottie.LottieAnimationView;

public class MainActivity extends AppCompatActivity {

  @Override
  protected void onCreate(Bundle savedInstanceState) {
    super.onCreate(savedInstanceState);
    setContentView(R.layout.activity_main);

    String url = "http://www.gs"
        + "tatic.co"
        + "m/play/points/i"
        + "cons/level"
        + "_badges/lot"
        + "tie_le"
        + "vel2.json";
    Log.i("MainActivity", "url is " + url);
    LottieAnimationView lottieAnimationView = findViewById(R.id.animation_view);

    lottieAnimationView.setAnimationFromUrl(url);
  }
}

File: kotlin/kotlin-and-android-development-featuring-jetpack/chapter-14-part-1/app/src/main/java/dev/mfazio/pennydrop/MainActivity.kt
package dev.mfazio.pennydrop

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import androidx.navigation.NavController
import androidx.navigation.fragment.NavHostFragment
import androidx.navigation.ui.setupWithNavController
import com.google.android.material.bottomnavigation.BottomNavigationView

class MainActivity : AppCompatActivity() {

    private lateinit var navController: NavController

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val navHostFragment = supportFragmentManager.findFragmentById(R.id.containerFragment) as NavHostFragment
        this.navController = navHostFragment.navController

        findViewById<BottomNavigationView>(R.id.bottom_nav)
            .setupWithNavController(this.navController)
    }
}
